### DISCUSSION TITLE
Welcome to Acer-Nitro-5-AN515-54-Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by itsmeshibintmz** [itsmeshibintmz/Acer-Nitro-5-AN515-54-Big-Sur#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gimmy52** [comment]
Hi, thank you to what have you done, i've tried to install Big Sur on my Nitro 5 515-54 by my self, but had some issue with GPIO, anyway with youf EFI i've succesfully installed Big Sur, but truckpad doesn't work and battery drain a lot 40% in 30 min web navigation, also with your last release.
 your trackpad ELAN o Synaptics?

**Reply by itsmeshibintmz** [reply]
> your trackpad ELAN o Synaptics?

Synaptics it is, Battery drain was okay for me compared to Windows.

The trackpad was too working for me, try clean "OC Clean Snapshot" and try again.
